qa/L0_storage_S3/test.sh

```diff
@@ -93,10 +93,11 @@ for ENV_VAR in "env" "env_dummy" "config"; do
 
     KIND="KIND_GPU"
 
-    for MAYBE_SLASH in "" "/"; do
+    # Test coverage for extra slashes 
+    for MAYBE_SLASH in "" "/" "//"; do
```

Comment:
Don't we also need to test with multiple internal slashes? I thought we had tests with the model repo at the top-level (just the bucket) and also more deeply nested...

Reply:
the addition of `// ` does add multiple internal slashes
The urls are now: 
```
s3://triton-bucket-1211
s3://triton-bucket-1211/models
s3://triton-bucket-1211/
s3://triton-bucket-1211//models/
s3://triton-bucket-1211//
s3://triton-bucket-1211///models//
```

Reply:
No we did not have tests that are deeper nested but when navigating the repo with the above s3 paths we are cleaning deeper urls as well since we use CleanPath in ParsePath as well